managers/tests/test_unit_managers_function_views.py

```diff
@@ -0,0 +1,74 @@
+import datetime
+
+from rest_framework.reverse import reverse
+from rest_framework.test import APIRequestFactory
+
+from django.test import TestCase
+
+from users.models import CustomUser
+from managers import views
+from managers.models import Project
+
+
+class DeleteProjectTests(TestCase):
+    def setUp(self):
+        self.user = CustomUser(
+            email="testuser@codepoets.it",
+            first_name='John',
+            last_name='Doe',
+            country='PL',
+            user_type=CustomUser.UserType.ADMIN.name,
+        )
+        self.user.password = 'newuserpasswd'
+        self.user.set_password(self.user.password)
+        self.user.full_clean()
+        self.user.save()
+
+        self.project = Project(
+            name='Example Project',
+            start_date=datetime.datetime.now().date() - datetime.timedelta(days=30),
+            stop_date=datetime.datetime.now().date(),
+            terminated=False,
+        )
+        self.project.full_clean()
+        self.project.save()
+        self.project.managers.add(self.user)
+        self.project.members.add(self.user)
+        self.url = reverse('custom-project-delete', args=(self.project.pk,))
```

Comment:
I think that it would be better to keep this url as a string

Reply:
Done